samples/msal-angular-v3-samples/angular15-sample-app/src/environments/environment.ts

```diff
@@ -3,7 +3,17 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  msalConfig: {
+      auth: {
+          clientId: '',
```

Comment:
nit: might use placeholders like "ENTER_CLIENT_ID" instead of empty strings here